### ISSUE TITLE
Algobasket is a scammer.

### ISSUE THREAD

**Issue by therealredvoid** [peatio/peatio#829]
BEWARE. 

Alot of people have had their bitcoins stolen by algobasket. I request admins to keep this on top to let the community be aware. If i saw this thread, i could have saved my money.

**Comment by peatioproject** [comment]
Yes, aggree. Algobasket wrote bad things for us too. One of our customers show us conversations screenshout. They still our customer 0.3 BTC and he didn't refund BTC.

Please don't send all money before work. And don't trust with anyone demo.

**Comment by dudetheman** [comment]
@vergeio is AlgoBasket.

**Comment by Thedabest** [comment]
> @vergeio is AlgoBasket.

yeah he clone to many account
